client/src/JobSelector.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { describe, expect, it, jest } from "@jest/globals";
+import { shallow } from "enzyme";
+import toJson from "enzyme-to-json";
+import JobSelector from "./JobSelector";
+
+const mockCategory = jest.fn();
+jest.mock(
+    "./Category",
+    () =>
+        function Category(props) {
+            mockCategory(props);
+            return <mock-Category />;
+        }
+);
+
+describe("JobSelector", () => {
+    // let data;
```

Comment:
There's an api call i intend to test with the categories and jobs.

Reply:
Do you want to finish this test before merging the PR? If not, would it be better to put in a TODO comment so we don't lose track of it later?

Reply:
I'll put it as TODO since it is dependent on the refactor of the AjaxHelper class.

Reply:
Made change.